Login.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Login</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+    <link rel="stylesheet" href="./public/styles/styles-login.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+    <div class="login-main-container"> 
+        <h3>Sign In to BSWJ</h3>
+        
+        <div class="login-icon-container">
+            <i class="fa fa-sign-in fa-5x" aria-hidden="true"></i>
+        </div>
+        <br>
```

Comment:
@lsimpson667 There are a few `<br>` in this document that are being used as a spacers. Consider using margins to give space. Use the br tag only when you need to break a line in a paragraph.